src/nvidia-modeset/src/nvkms-rm.c

```diff
@@ -4983,7 +4983,7 @@ static void MuxInit(const NVDpyEvoRec *pDpyEvo)
          * is disconnected from the GPU. */
 
         /* Map with hard-coded data for systems known to support dynamic mux
-         * switching. This is a poor-man's alternative to the WDDM driver's
+         * switching. This is a poor-person's alternative to the WDDM driver's
```

Comment:
```suggestion
         * switching. This is a poor-human's alternative to the WDDM driver's
```

Reply:
You can't be such disrespectful against non-human persons!

Reply:
```suggestion
         * switching. This is a poverty-stricken person's alternative to the WDDM driver's
```

Reply:
Perhaps financially-disadvantaged?